### ISSUE TITLE
Segmentation fault on import torch_geometric 

### ISSUE THREAD

**Issue by midannii** [pyg-team/pytorch_geometric#6878]
### 🐛 Describe the bug

When I try to import torch_geometric I get a segmentation fault error.

I am just trying to import torch_geometric to get started and as such haven't even used any functions yet, since every time I try to import I get this error.

(I can import `torch_scatter`, `torch_cluster`, and `torch_sparse`)

Help me plz.. 

### Environment

* PyG version: latest
* PyTorch version: 1.12.1
* OS: linux(centOS)
* Python version: 3.8.16 
* CUDA/cuDNN version: 11.2
* How you installed PyTorch and PyG (`conda`, `pip`, source): 
 - pytorch: `conda install pytorch==1.12.1 torchvision==0.13.1 torchaudio==0.12.1 cudatoolkit=10.2 -c pytorch`
 - pyg: `conda install pyg -c pyg`
* Any other relevant information (*e.g.*, version of `torch-scatter`):
- torch_scatter: 2.1.0
- torch_cluster: 1.6.0
- torch_sparse: 0.6.16
- torch: 1.9.1 (checked in `conda list`)
- torch_spline_conv: 1.2.1 (checked in `conda list`)

**Comment by EdisonLeeeee** [comment]
It appears that you are using different versions of PyTorch simultaneously, such as 1.9.1 and 1.12.1. Would you be able to uninstall one of them and try again?

**Comment by EulerPascal404** [comment]
> PyTorch version: 1.12.1
> torch: 1.9.1 (checked in `conda list`)

Yup, it seems you have 2 different versions of pytorch installed. You can uninstall pytorch with the following command: `conda uninstall pytorch`. Then try reinstalling pytorch - it should fix the error.
